### PR TITLE
JBTM-3507 Changing old links in README

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/JBTM-3507

NO_TEST

### CODE REVIEW

**Review by ochaloup — APPROVED** [review]
Seems fine to me.

File: README.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ To get your developer life easier use the checkstyle plugins for your IDE
   file https://github.com/wildfly/wildfly-checkstyle-config
 * install checkstyle plugin to your favourite IDE
     - IntelliJ IDEA: https://plugins.jetbrains.com/plugin/1065-checkstyle-idea
-    - Eclipse: http://eclipse-cs.sourceforge.net
+    - Eclipse: https://eclipse-cs.sourceforge.net
```
Comment (mmusgrov) [inline README.md:142]: This should be https://checkstyle.org/eclipse-cs/#!/

File: README.md [hunk 2]
```diff
@@ -161,9 +161,9 @@ JTA and JTA in turn a superset of Core.
 
 3rd party dependency management is done via maven. Note that versions of most 3rd party components are resolved via the JBossAS component-matrix
 pom.xml, even when building standalone releases. The version of JBossAS to use is determined by the top level pom.xml
-You may need to set up maven to use the jboss.org repositories: http://community.jboss.org/wiki/MavenGettingStarted-Users
+You may need to set up maven to use the jboss.org repositories: https://community.jboss.org/wiki/MavenGettingStarted-Users
 
 Maven is provided in the tools/maven section, though later versions of this may work. Download locations are:
```
Comment (mmusgrov) [inline README.md:166]: Ondra has just [submitted a PR](https://github.com/jbosstm/narayana/pull/1868), which is ready to be merged, that removes maven from our tools section and just needs merging. So I think this paragraph can be deleted.

**Review by mmusgrov — CHANGES_REQUESTED** [review]
Good catch. I did have two change requests.

File: README.md (hunk 2 above)
Comment (ochaloup) [reply README.md:166]: +1, I've just missed this fact :-)

Comment (RanabirChakraborty) [reply README.md]: Thanks @mmusgrov done the changes :) check once

**Review by mmusgrov — APPROVED** [review]
Thanks very much. We look forward to your next fix :-)

**Pull request merged**